scripts/api-docs-builder/src/hookHandler.ts

```diff
@@ -1,5 +1,5 @@
 import * as tae from 'typescript-api-extractor';
-import { formatProperties, formatParameters, formatType } from './formatter';
+import { formatProperties, formatParameters, formatType } from './formatter.js';
```

Comment:
`nodenext` needs file extensions. I figured there won't be a huge number of files in this package.